hasher-matcher-actioner/src/OpenMediaMatch/templates/components/banks/banks_grid.html.j2

```diff
@@ -0,0 +1,121 @@
+{# Copyright (c) Meta Platforms, Inc. and affiliates. #}
+<h1>Banks</h1>
+<p class="text-secondary">Banks are like folders. You can do the following on this page:</p>
+<ul class="text-secondary">
+    <li>Create new banks</li>
+    <li>Add content: Add content to each bank</li>
+    <li>Search by content: Find banks that contain a certain piece of content</li>
+</ul>
+{% include 'components/banks/actions_bar.html.j2' %}
+
+<div class="row row-cols-1 row-cols-md-4 g-4">
+    <table class="table table-hover">
+        <thead>
+            <tr>
+                <th scope="col">#</th>
+                <th scope="col">Bank Name</th>
+                <th scope="col">Matching Enabled Ratio</th>
+                <th scope="col">Actions</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for bank in bankList %}
+            <tr id="bank_item_{{bank['name']}}">
+                <th scope="row" style="background-color: transparent;">{{ loop.index }}</th>
+                <td style="background-color: transparent;">{{ bank['name'] }}</td>
+                <td style="background-color: transparent;">{{ bank['matching_enabled_ratio'] }}</td>
+                <td style="background-color: transparent;">
+                    {% with bank_title=bank['name'] %}
+                        {% include 'components/banks/add_to_bank_form.html.j2' %}
+                    {% endwith %}
+                </td>
+            </tr>
```

Comment:
eventually, these will link to their own page with stats and such.

Leaving that up to ya'll too